dotnetv3/Dockerfile

```
@@ -3,13 +3,15 @@
 # GA updates: https://github.com/awsdocs/aws-doc-sdk-examples/issues/4126 
 FROM mcr.microsoft.com/dotnet/sdk:6.0
 
-#uncomment these lines to also enable the .NET 5.0 SDK if running 5.0 examples:
-#COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/sdk /usr/share/dotnet/sdk
-#COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/shared /usr/share/dotnet/shared
+# Enable the .NET 5.0 SDK if running 5.0 examples:
+COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/sdk /usr/share/dotnet/sdk
+COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/shared /usr/share/dotnet/shared
 
-#copy the dotnetv3 folder. Navigate to a service folder and/or a scenario folder,
-#such as dotnetv3/CloudWatch/Scenarios to build and run an interactive scenario with `dotnet run`
+# Copy the dotnetv3 folder and run unit tests from the top level solution.
+# Or navigate to a service folder or a scenario folder,
+# such as dotnetv3/CloudWatch/Scenarios to build and run an interactive scenario with `dotnet run`
 COPY . /dotnetv3
+WORKDIR /dotnetv3
```

Alas, `WORKDIR` which is a best practice Docker convention, does not work with GitHub Actions. Also, you need to build.

```suggestion
RUN cd /dotnetv3 && dotnet build
```

---

I originally had RUN here, but it never worked correctly.

---

Ok, it works for the building but not for the test command, so I added the directory to my test instructions.